hr_attendance_hour_type_report/models/hr_attendance.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import datetime as dt
+import logging
+
+import pytz
+
+from odoo import _, api, exceptions, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    worked_hours_nighttime = fields.Float(
+        string="Night hours", compute="_compute_worked_hours_night", store=True
+    )
+    worked_hours_daytime = fields.Float(
+        string="Day hours",
+        compute="_compute_worked_hours_night",
```

Comment:
```suggestion
        compute="_compute_worked_hours",
```
just a proposal as this method name in relation with field name is a bit confusing